.github/workflows/docker-test.yml

```diff
@@ -0,0 +1,28 @@
+name: Docker Tests
+
+on:
+  workflow_call:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build Docker image
+        run: docker build -t alma-nv-test .
+      - name: Test Docker entrypoint script
+        run: |
+          # Test that the container starts and shows help without requiring privileged mode
+          docker run --rm alma-nv-test --help
+      - name: Validate Docker script syntax
+        run: |
+          # Check that docker-entrypoint.sh has valid bash syntax
+          bash -n docker-entrypoint.sh
+          # Check that run-alma.sh has valid bash syntax
+          bash -n run-alma.sh
+      - name: Test Docker build process
+        run: |
+          # Verify the alma binary was built and copied correctly
+          docker run --rm alma-nv-test which alma
```

Comment:
I think we can remove this line - the entry-point is already `alma`, so if it is missing the entry point will fail anyway.